tests/test_git_archive_file.py

```diff
@@ -28,8 +28,8 @@ def test_archive_file_on_any_not_tags_branch_without_default_branch(self, temp_r
 
     def test_archive_file_on_any_not_tags_branch_with_default_branch(self, temp_repo):
         git = temp_repo.get_repo_git()
-        git.checkout("master")
-        git.config("git-extras.default-branch", "master")
+        git.checkout("main")
```

Comment:
If I am correct, I think the branch here is set to master intently. Because the current branch of the  `temp_repo.get_repo_git()` seems to be already `main`.

@vanpipy 
What do you think?

Reply:
Oh I think I know what this is - I have customized `init.defaultBranch` to be `main`. Which is why I had to make that change. I'll fix the name of the branches used in the test to be less confusing